webapp/src/containers/addOrEditSubscriptions/subComponents/filtersPanel.tsx

```diff
@@ -127,6 +140,11 @@ const FiltersPanel = forwardRef<HTMLDivElement, FiltersPanelProps>(({
         return {isLoading, data: data as FieldsFilterData[], isError, error: (apiErr as FetchBaseQueryError)?.data as APIError | undefined};
     };
 
+    const getItems = (payload: SearchFilterItemsParams | null) => {
+        const {isLoading, data: getItemsData, isError, isSuccess, error: apiErr} = getApiState(Constants.pluginApiServiceConfigs.getFilterData.apiServiceName, payload);
+        return {isLoading, data: getItemsData as FieldsFilterData[], isError, isSuccess, error: (apiErr as FetchBaseQueryError)?.data as APIError | undefined};
```

Comment:
```suggestion
        return {isLoading, data: getItemsData as FilterFieldsData[], isError, isSuccess, error: (apiErr as FetchBaseQueryError)?.data as APIError | undefined};
```
I think `FilterFieldsData` is more appropriate name for the type.